indra/llkdu/llimagej2ckdu.cpp

```diff
@@ -579,7 +579,7 @@ bool LLImageJ2CKDU::decodeImpl(LLImageJ2C &base, LLImageRaw &raw_image, F32 deco
 															mCodeStreamp.get()));
 				}
 				// Do the actual processing
-				F32 remaining_time = limit_time ? static_cast<F32>(decode_time - decode_timer.getElapsedTimeF32()) : 0.0f;
+				F32 remaining_time = limit_time ? decode_time - decode_timer.getElapsedTimeF32()() : 0.0f;
```

Comment:
```suggestion
				F32 remaining_time = 0.0f;
				if (limit_time)
					remaining_time = decode_time - decode_timer.getElapsedTimeF32();
```

Reply:
Using 3 lines instead of 1 doesn't help to see the whole algorithm
We'd better hide these simple details to allow user see the picture

Reply:
It's not about counting the lines. 
Ternaries are fine, but if we need to add more operators in order to use them, it's better to avoid that.

Reply:
Agree with marchcat, and personaly I find longer Ternaries slower to read than clear if+else. They are convinient for some cases and easy to read when very short, but if they require either more code to make work, too long or use other operations inside it's probably better to avoif them.